src/status_im/ui/screens/keycard/onboarding/views.cljs

```diff
@@ -65,73 +65,78 @@
-                                    (i18n/label :t/keycard-onboarding-start-step1-text)]
-                                   ["2"
-                                    (i18n/label :t/keycard-onboarding-start-step2)
-                                    (i18n/label :t/keycard-onboarding-start-step2-text)]
-                                   ["3"
-                                    (i18n/label :t/keycard-onboarding-start-step3)
-                                    (i18n/label :t/keycard-onboarding-start-step3-text)]]]
-         ^{:key number} [react/view {:flex-direction :row
-                                     :margin-top     15}
-                         [react/view {:border-width    1
-                                      :border-radius   20
-                                      :border-color    colors/gray-light
-                                      :align-items     :center
-                                      :justify-content :center
-                                      :width           40
-                                      :height          40}
-                          [react/text {:style {:typography :title}}
-                           number]]
-                         [react/view {:align-items     :flex-start
-                                      :justify-content :flex-start
-                                      :margin-left     11}
-                          [react/view
-                           [react/text {:style {:typography :main-medium}}
-                            header]]
-                          [react/view
-                           [react/text {:style {:color         colors/gray
-                                                :padding-right 35}}
-                            text]]]])]]
-     [react/view {:margin-bottom   12
-                  :align-items     :center
-                  :justify-content :center}
-      [react/image {:source      (resources/get-image :keycard-phone)
-                    :resize-mode :center
-                    :style       {:width  160
-                                  :height 170}}]]]]])
+(defview start []
+  (letsubs [flow [:hardwallet-flow]]
+    [react/view styles/container
+     [toolbar/toolbar
+      {:transparent? true
+       :style        {:margin-top 32}}
+      toolbar/default-nav-back
+      nil]
+     [react/scroll-view
+      [react/view {:flex            1
+                   :flex-direction  :column
+                   :justify-content :space-between
+                   :align-items     :center}
+       [react/view {:flex-direction :column
+                    :align-items    :center}
+        [react/view {:margin-top 16}
+         [react/text {:style {:typography :header
+                              :text-align :center}}
+          (i18n/label :t/keycard-onboarding-start-header)]]
+        [react/view {:margin-top 16
+                     :width      311}
+         [react/text {:style {:font-size   15
```

Comment:
I think `:font-size 15` is a default typography style, that uses line-height=21. So probably these 2 settings can be removed altogether.